src/app/src/main/kotlin/org/vaccineimpact/api/app/ResultRedirector.kt

```diff
@@ -0,0 +1,60 @@
+package org.vaccineimpact.api.app
```

Comment:
This code largely duplicates existing logic in `OneTimeLinkController`, but since that will be removed soon I didn't try and make them share code.